mgc/terraform-provider-mgc/Makefile

```diff
@@ -107,4 +107,10 @@ setup-config: ## Setup configuration file for Terraform or OpenTofu
 	@echo "File $$config_file written to $$HOME:"
 	@cat $$config_file
 
+build: ## Build the plugin
```

Comment:
Nice!
We can be more specific here and call it `build-tf` or something similar.
Also we can leverage gorelease, with some `goreleaser build --snapshot --clean --single-target --config=goreleaser_internal.yaml`, which builds it according to plan and uses proper version number (algo is user-platform-specific)

Reply:
Thanks for the review and suggestions! About the goreleaser, i didn't know about it, thank you for the tip.
As it is installed locally, what you think would be the best way to integrated with the script? Document as a pre-requisite or use the [bash script](https://goreleaser.com/install/#bash-script)?